LadderBots.json

```diff
@@ -0,0 +1,24 @@
+{
+	"Bots": {
+		"ProtossBot": {
+			"Race": "Protoss",		#Race of your bot
```

Comment:
This is not good. If you want to add comments to a JSON structure (which doesn't specify any commentary symbol actually) you have to use // or /* i.e. follow native JavaScript style. # is from other languages.

Reply:
I will update to the latest version of rapidjson and test it with that.  I'll try to do that tonight, but I think it would be better overall to keep comments out of the json file if possible, as it may confuse some parsers